test/test_util.py

```diff
@@ -9,3 +9,6 @@ class TestUtil(unittest.TestCase):
 
     def test_inpath(self):
         self.assertTrue(inpath('python'))
+
+     def test_not_inpath(self):
+        self.assertFalse(inpath('testing'))
```

Comment:
Can we make sure that the test is super extra unlikely to ever pass on anyone's system?  I'm thinking it's possible that a developer will have a "testing" executable in their path, so maybe we could make it "cve_bin_tool_test_for_not_in_path" or something equally hideous, ridiculous, and unlikely to ever pass.